upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -880,6 +880,43 @@ else
     echo "====> ${state_name} has been completed" | tee -a "${LOG_FILE}"
 fi
 
+state_name="UPGRADE_CERTMANAGER_CHART"
+state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
+if [[ ${state_recorded} == "0" && $(hostname) == "${PRIMARY_NODE}" ]]; then
+    echo "====> ${state_name} ..." | tee -a "${LOG_FILE}"
+    {
+        # Only remove these charts if installed
+        if helm list -n cert-manager --filter cray-certmanager | grep 'cray-certmanager$' > /dev/null 2>&1; then
+            helm uninstall -n cray-certmanager cray-certmanager
+        fi
+        if helm list -n cray-certmanager-init --filter cray-certmanager-init | grep cray-certmanager-init > /dev/null 2>&1; then
+          helm uninstall -n cray-certmanager-init cray-certmanager-init
+        fi
+        tmp_manifest=/tmp/certmanager-tmp-manifest.yaml
+
+        cat > "${tmp_manifest}" <<EOF
+apiVersion: manifests/v1beta1
+metadata:
+  name: cray-certmanager-images-tmp-manifest
+spec:
+  charts:
+  -
+EOF
+
+        yq r "${CSM_MANIFESTS_DIR}/platform.yaml" 'spec.charts.(name==cray-drydock)' | sed 's/^/    /' >> "${tmp_manifest}"
+        yq r "${CSM_MANIFESTS_DIR}/platform.yaml" 'spec.charts.(name==cray-certmanager)' | sed 's/^/    /' >> "${tmp_manifest}"
```

Comment:
The hyphens aren't ending up right.   This is what I get:
```
apiVersion: manifests/v1beta1
metadata:
  name: cray-certmanager-images-tmp-manifest
spec:
  charts:
  -
    name: cray-drydock
    source: csm-algol60
    version: 2.18.0
    namespace: loftsman
    name: cray-certmanager
    source: csm-algol60
    version: 0.7.2
    namespace: cert-manager
```

Reply:
@mitchty can you use `yq4` instead of `yq`? It will help us switch from yq version 3.